cl/recap/mergers.py

```diff
@@ -1457,7 +1457,11 @@ async def clean_duplicate_attachment_entries(
         for attachment in attachment_dicts:
             attachment_number = attachment["attachment_number"]
             pacer_doc_id = attachment["pacer_doc_id"]
-            if dupe.pacer_doc_id == pacer_doc_id:
+            acms_document_guid = attachment["acms_document_guid"]
```

Comment:
Won't this throw a keyerror if `acms_document_guid` is not present in the attachment dict?

I'm thinking...we shouldn't have ACMS duplicates at the moment right? So maybe we should just skip calling `clean_duplicate_attachment_entries` for ACMS attachment pages entirely?

Reply:
> Won't this throw a keyerror if acms_document_guid is not present in the attachment dict?

You're correct! Thank you so much for pointing out this issue. I will make sure to correct it immediately

> we shouldn't have ACMS duplicates at the moment right? 

You're right. ACMS is a new source of data. 

> So maybe we should just skip calling clean_duplicate_attachment_entries for ACMS attachment pages entirely?

This is a great idea and We can use the `is_acms_attachment` parameter to skip the step.